.circleci/ds210_fasttrack_outputs.txt

```diff
@@ -29,5 +29,6 @@ fmriprep/sub-02/func/sub-02_task-cuedSGT_run-01_space-MNI152NLin2009cAsym_desc-b
 fmriprep/sub-02/func/sub-02_task-cuedSGT_run-01_space-MNI152NLin2009cAsym_desc-brain_mask.nii.gz
 fmriprep/sub-02/func/sub-02_task-cuedSGT_run-01_space-MNI152NLin2009cAsym_desc-preproc_bold.json
 fmriprep/sub-02/func/sub-02_task-cuedSGT_run-01_space-MNI152NLin2009cAsym_desc-preproc_bold.nii.gz
+fmriprep/sub-02/func/sub-02_task-cuedSGT_run-01_space-MNI152NLin2009cAsym_desc-tedana_T2starmap.nii.gz
```

Comment:
By metadata, do you mean specific fields for the sidecar JSON file, or general documentation about this derivative?

Reply:
Sidecar fields.

Reply:
T2* maps _must_ be in seconds, but it might be nice to have a `Units` field anyway. [The qMRI appendix](https://bids-specification.readthedocs.io/en/stable/99-appendices/11-qmri.html#metadata-requirements-for-qmri-maps) also recommends including `BasedOn`, `EstimationReference`, and `EstimationAlgorithm` fields. 

I think fMRIPrep covers `BasedOn` already, but `EstimationReference` and `EstimationAlgorithm` might be useful to include. I think @handwerkerd would be able to say what those fields should be set to better than me though.

Reply:
Not sure how much detail can go into `EstimationAlgorithm` but a long option would be:
"Fit to monoexponential T2* decay model [across the mean value across time for each voxel and echo]"
We don't explicitly reference anything. My understanding is the math could go back to some really early NMR papers. I I think the first paper to average across time for multi-echo data is: Poser et al "BOLD contrast sensitivity enhancement and artifact reduction with multiecho EPI: parallel-acquired inhomogeneity-desensitized fMRI" MRM 2006 https://doi.org/10.1002/mrm.20900

Reply:
fMRIPrep could hypothetically support multiple t2smap options (e.g., see #2728), but for the current implementation a short algorithm description might be "nonlinear monoexponential".

Reply:
To be clear, I don't think we need a "first paper", I think we need a meaningful reference that would make the actual algorithm clear. If it makes sense to use one of the citations on https://tedana.readthedocs.io/, I'm happy to do that.